Render jobs from templates (#1363)
kyma-project/test-infra: docs/prow/prow-jobs.md

## Create jobs

> Jobs should be generated from templates in `templates` directory. For details on how to do that see [Configure templates](./templates.md).

Reviewer: ```suggestion
Generate jobs from templates in the `templates` directory. For details on how to configure templates, see [Configure templates](./templates.md).
```

Author: I resolved it differnetly.
